ui/src/app/workflows/components/workflow-logs-viewer/workflow-logs-viewer.tsx

```diff
@@ -297,7 +297,7 @@ export function WorkflowLogsViewer({workflow, nodeId, initialPodName, container,
                 )}
                 {execSpec(workflow).podGC && (
                     <>
-                        <WarningIcon /> Your pod GC settings will delete pods and their logs immediately on completion.
+                        <WarningIcon /> Your pod GC settings will delete pods and their logs {execSpec(workflow).podGC.deleteDelayDuration ? `after ${execSpec(workflow).podGC.deleteDelayDuration}` : "immediately"} on completion.
```

Comment:
thought this might happen -- the line is too long now so lint is [failing](https://github.com/argoproj/argo-workflows/actions/runs/8251525284/job/22568803086?pr=12784)

```suggestion
                        <WarningIcon /> Your pod GC settings will delete pods and their logs{' '}
                        {execSpec(workflow).podGC.deleteDelayDuration ? `after ${execSpec(workflow).podGC.deleteDelayDuration}` : 'immediately'} on completion.
```